io.github.quotient_im.Quaternion.yaml

```diff
@@ -0,0 +1,66 @@
+id: com.github.quaternion
+rename-icon: quaternion
+runtime: org.kde.Platform
+runtime-version: '6.7'
+sdk: org.kde.Sdk
+command: quaternion
+finish-args:
+- --share=ipc
+- --share=network
+- --socket=wayland
+- --socket=fallback-x11
+- --device=dri
+- --filesystem=xdg-download
+- --talk-name=org.freedesktop.secrets
+- --talk-name=org.kde.kwalletd5
+- --talk-name=org.freedesktop.Notifications
+- --talk-name=org.kde.StatusNotifierWatcher
+modules:
+- name: libolm
+  buildsystem: cmake-ninja
+  sources:
+  - type: git
+    url: https://gitlab.matrix.org/matrix-org/olm.git
+    tag:  '3.2.15'
+  config-opts:
+  - -DBUILD_SHARED_LIBS=OFF
+  cleanup: [ /include, /lib, /share ]    
```

Comment:
```suggestion
  cleanup:
    - /include
    - /lib
    - /share    
```

please use proper YAML syntax.

Reply:
same applies below.

Reply:
It is proper YAML. But I understood that you request to use the block syntax instead of the flow syntax.

Reply:
this should be part of a toplevel cleanup rule so that you don't need to repeat it for every module.

Reply:
/lib/pkgconfig, /share/man too.